packages/vuetify/src/components/VForm/VForm.ts

```diff
@@ -132,6 +132,7 @@ export default mixins(
       },
       on: {
         submit: (e: Event) => this.$emit('submit', e),
+        reset: (e: Event) => this.$emit('reset', e),
```

Comment:
maybe just pass all listeners?

Reply:
Ok, I've changed this to pass on all `listeners$`, but other than `submit` and `reset` I've only found `onformdata` event that is passed (https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement), but I couldn't find a way to test that event.

Reply:
all general dom events (click, hover, etc) are also passed
regarding test - i guess you can take the DOM element, trigger any event and check if the component's listener has been called

Reply:
I've added a couple of tests (for `click` and `hover`) that should cover general DOM events. If you think I need to cover more of them let me know, it should be easy to add by calling `expectDomEventListernerCalled(<new event name>)`.